packages/devtools_app/test/support/mocks.dart

```diff
@@ -34,7 +36,18 @@ class MockServiceManager extends Mock implements ServiceConnectionManager {
       FakeServiceExtensionManager();
 }
 
-class MockVmService extends Mock implements VmServiceWrapper {}
+class MockVmService extends Mock implements VmServiceWrapper {
```

Comment:
If this logic is only needed in the timeline test, consider providing it there with mockito.

Reply:
This is also used in vm_flag_widgets_test